### ISSUE TITLE
vim_run_command_exterm() doesn't work when command ending with a new line?

### ISSUE THREAD

**Issue by saidelike** [hands-free-vim/neovim-talon#10]
this does nothing and doesn't work:
```
tab reset:                  user.vim_run_command_exterm(":TabooReset\n")
```

this show the command and then when we hit enter it actually works:
```
tab reset:                  user.vim_run_command_exterm(":TabooReset")
```

**Comment by fidgetingbits** [comment]
You probably have to escape the \n ? I'm pretty sure I have code that works for doing this in talon-vim somewhere

**Comment by saidelike** [comment]
yes I'm pretty sure it used to work, but I don't see any escape in your talon-vim. I've created the issue so we can try to understand why

**Comment by saidelike** [comment]
It seems it's only this command that doesn't work with a new line, but another other commands like the lazy ones work fine with a new line
